src/Nri/Ui/Modal/V11.elm

```diff
@@ -425,16 +426,20 @@ modalStyles =
     -- Border
     , borderRadius (px 20)
     , boxShadow5 zero (px 1) (px 10) zero (rgba 0 0 0 0.35)
+    , Css.Media.withMedia
+        [ Css.Media.all [ Css.Media.maxWidth (px 1000) ] ]
+        [ borderRadius zero
+        ]
 
     -- Spacing
-    , margin2 (px 50) auto
+    , margin2 (px 20) auto
 
     -- Size
     , width (px 600)
     , backgroundColor Colors.white
 
     -- the modal should grow up to the viewport minus a 50px margin
```

Comment:
Maybe we should update this comment too?

Reply:
Good catch, updated.